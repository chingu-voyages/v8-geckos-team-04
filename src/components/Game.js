import React from 'react';
import Video from './Video';
import { useGlobal } from 'reactn';

function Game() {
  const [global] = useGlobal();

  function newGame(){
    window.location.reload(); // refresh the page
  }

  return(
    <div>
      <h1><a href='/'>Guess The Language</a></h1>
        <Video />
        <div className='score-box'>
          Question: {global.qNum}/10<br />
          Score: {global.score}
        </div>
        <button className='new-game-btn' onClick={newGame}>New Game</button>
    </div>
  )
} 

export default Game;