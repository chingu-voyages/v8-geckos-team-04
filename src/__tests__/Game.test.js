import React from 'react';
import { create } from 'react-test-renderer';
import Game from '../components/Game';

describe('Main', () => {
  test('it matches snapshot', () => {
    const component = create(<Game />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});

