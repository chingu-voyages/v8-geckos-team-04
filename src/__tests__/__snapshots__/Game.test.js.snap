// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main it matches snapshot 1`] = `
<div
  className="sidebars"
>
  <h1>
    <a
      href="/"
    >
      Guess The Language
    </a>
  </h1>
  <div
    className="score-box"
  >
    <div
      className="question"
    >
      Question: 
      /10
    </div>
    <div
      className="score"
    >
      Score: 
    </div>
  </div>
  <div
    className="video"
  >
    <div
      className="youtube-video-wrapper"
    >
      <div
        className="title-blocker"
      >
        <img
          alt=""
          id="imagehidestitle"
          src="./images/titlecover.jpg"
        />
      </div>
      <div
        id="whatlanguage"
      >
        What language do you think it is?
      </div>
      <iframe
        className="youtube-video-iframe"
        height="349"
        id={0}
        src="https://www.youtube.com/embed/3FGc0zaIg2k?start=5&end=120&autoplay=1&mute=1"
        title="English"
        width="560"
      />
    </div>
    <div
      className="choices"
    >
      <button
        disabled={false}
        id="French"
        onClick={[Function]}
      >
        French
      </button>
      <button
        disabled={false}
        id="English"
        onClick={[Function]}
      >
        English
      </button>
      <button
        disabled={false}
        id="Italian"
        onClick={[Function]}
      >
        Italian
      </button>
    </div>
    <div
      className="feedback"
    />
  </div>
  <div
    className="new-game-btn-wrapper"
  >
    <button
      className="new-game-btn"
      onClick={[Function]}
    >
      New Game
    </button>
  </div>
</div>
`;
